src/scripts/main.js

```diff
@@ -1,5 +1,33 @@
 'use strict';
 
 document.addEventListener('click', (e) => {
-  // write code here
+  const targetDiv = e.target.closest('div');
+
+  if (targetDiv.tagName === 'DIV') {
```

Comment:
Consider checking if `targetDiv` is not null before checking its `tagName`. This will prevent potential runtime errors if `closest` returns null.